lab-kyle/gulpfile.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
```

Comment:
Great work!  In addition to the lib and test folders you may want to explicitly add the server.js file as well :)